### PR TITLE
V0.3

### PR DESCRIPTION
**Pull request by JarlDysnomia**
### New compositions and replaced.

**NATO Composition:**
Repair Station

NATO FOB

NATO Heavy Artillery

Tank Position

**CTRG Composition:**
CTRG Camp

CTRG FOB

CTRG Artillery

**Syndikat Composition:**
Syndikat Camp

Syndikat Execution ground

**General Composition**

POW Camp.

**CSAT Composition**
CSAT QRF

CSAT Repair Station

CSAT Tank Position

**AAF Composition**

AAF Repair Station

AAF Tank Position

AAF Artillery

**Civilian Composition:**

Civilian Market

### CODE REVIEW

File: Source/JCC/config.cpp [hunk 1]
```diff
@@ -206,6 +873,115 @@ class CfgPatches
...
+					class Object29 {side=8;vehicle="Land_ClothesLine_01_full_F";rank="";position[]={9.44824,-5.72412,-0.00143862};dir=0;};
+					class Object30 {side=8;vehicle="Box_Syndicate_Ammo_F";rank="";position[]={2.57813,-10.9834,-0.00243998};dir=309.632;};
+					class Object31 {side=8;vehicle="Land_PaperBox_01_small_closed_white_med_F";rank="";position[]={-4.00537,10.9072,-0.00143862};dir=94.3978;};
+					class Object32 {side=8;vehicle="Box_Syndicate_WpsLaunch_F";rank="";position[]={3.39111,-11.3301,-0.00243902};dir=266.449;};
+					class Object33 {side=8;vehicle="Box_Syndicate_Ammo_F";rank="";position[]={-0.427734,-11.8804,-0.00243902};dir=3.19189e-006;};
+					class Object34 {side=8;vehicle="Box_Syndicate_Ammo_F";rank="";position[]={0.472168,-11.896,-0.0024395};dir=360;};
+					class Object35 {side=8;vehicle="Box_Syndicate_Wps_F";rank="";position[]={1.48828,-11.9165,-0.00243855};dir=360;};
+					class Object36 {side=8;vehicle="Box_Syndicate_Wps_F";rank="";position[]={2.54053,-11.8994,0.0157061};dir=360;};
+				};
+				class syndikatExecution
```
Comment (Radiumio) [inline Source/JCC/config.cpp]: What is "Syndikat Execution"?

Comment (JarlDysnomia) [reply Source/JCC/config.cpp]: > What is "Syndikat Execution"?

Execution ground for well people who might want to use that, i just didn't know where to put it under so i put it under syndikat.

Comment (Radiumio) [reply Source/JCC/config.cpp]: Okay, but maybe a better name for it? Can you send a picture of what it looks like?

Comment (JarlDysnomia) [reply Source/JCC/config.cpp]: > Okay, but maybe a better name for it? Can you send a picture of what it looks like?

Sure give me a min

**Comment by Radiumio** [comment]
Do you think we can we change `transmitter1` to something else? I don't like the generic naming of it, it doesn't fit in well with the names of all your other compositions.

```sqf
class transmitter1
				{
					name = "Transmitter 1";
```

to...

```sqf
class civilianTransmitterPost
				{
					name = "Civilian Transmitter Post";
```
Maybe?

File: Source/JCC/config.cpp (hunk 1 above)
Comment (Radiumio) [reply Source/JCC/config.cpp]: `syndikatExecutionEnclosure`?

**Review by Radiumio — APPROVED** [review]

**Pull request merged**